src/Admin/GoogleFonts.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Class GoogleFonts.
+ *
+ * Registers Google fonts for admin screens.
+ *
+ * @since 1.6.0
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Enqueue Google Fonts stylesheet.
+ *
+ * @since 1.6.0
+ */
+final class GoogleFonts implements Conditional, Delayed, Service, Registerable {
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		return is_admin() && ! wp_doing_ajax();
+	}
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'admin_enqueue_scripts';
+	}
+
+	/**
+	 * Provides the asset handle.
+	 *
+	 * @return string
+	 */
+	public function get_handle() {
+		return 'amp-admin-google-fonts';
+	}
+
+	/**
+	 * Runs on instantiation.
+	 */
+	public function register() {
+		// PHPCS ignore reason: WP will strip multiple `family` args from the Google fonts URL while adding the version string,
+		// so we need to avoid specifying a version at all.
+		wp_register_style( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
```

Comment:
If `wp_default_styles` action were used, then this would need to grab the `WP_Styles` instance passed to the callback for the `wp_default_styles` action. And then it would mean `register` couldn't be used as-is here.

Reply:
Done in 81b9a88aa